### ISSUE TITLE
yolov4-mish.cfg add `stopbackward=800`

### ISSUE THREAD

**Issue by Linchunhui** [AlexeyAB/darknet#6993]
Why add this line?

**Comment by erikguo** [comment]
I have the same confusion. I check the code network.c. This "stopbackward=800' means stop backward after 800 iterations under GPU environment. 

Should we remove this line when do training custom data ?  Thank you

@AlexeyAB

**Comment by AlexeyAB** [comment]
> This "stopbackward=800' means stop backward **after** 800 iterations under GPU environment.

No. "stopbackward=800' means stop backward **before** 800 iterations.
You shouldn't change it.

**Comment by erikguo** [comment]
OK. Thanx. I made a misunderstanding :(.
